### DISCUSSION TITLE
cp editor 能否支持调试

### DISCUSSION THREAD

**Discussion by zjzj1206** [cpeditor/cpeditor-cn#10] [提问] [answered]
写代码时常常出错，但是又不能调试，cp editor 能不能支持，或者以后是否有可能

**Comment by ouuan** [comment, marked as answer]
现在可以用 cerr 进行输出调试: https://cpeditor.org/zh/docs/tips/#%E4%BD%BF%E7%94%A8-stderr-%E8%BF%9B%E8%A1%8C%E8%B0%83%E8%AF%95

也可以通过 gdb 调试，右键标签栏可以打开可执行文件所在文件夹。

相关 issues: https://github.com/cpeditor/cpeditor/issues/17 https://github.com/cpeditor/cpeditor/issues/919
